uint8 a = 1
int8 b = 2
uint16 c = 3
int16 d = 4
uint32 e = 5
int32 f = 6
uint64 g = 7
int64 h = 8
float32 fa = 1.5
float64 fb = 40.9
string str = hello there
string str2 = this string has "quotes" and \slashes\ in it